system/Database/SQLSRV/Forge.php

```diff
@@ -218,10 +237,8 @@ protected function _dropIndex(string $table, object $indexData)
 
     /**
      * Process indexes
-     *
-     * @return array|string
      */
```

Comment:
Added second parameter $asQuery. This tells the function to return a stand alone query rather than the partial sql used when creating a table. Also returns array instead of string.